release-notes-5.0.0.md

```diff
@@ -0,0 +1,14 @@
+!not-ready-for-release!
+
+#### Version Number
+${version-number}
+
+#### New Features
+
+#### Known Issues
+
+#### Breaking Changes
+* [SCMOD-10362](https://portal.digitalsafe.net/browse/SCMOD-10362): Migrate from Nashorn to Graal.JS  
+The Nashorn engine (scheduled for removal in [JEP372](https://openjdk.java.net/jeps/372)) has been deprecated and will 
+be replaced with the GraalVM JavaScript engine. This necessitated a number of changes to the workflow control script 
+that are incompatible with workers built on a worker-document framework version older than 4.5.0.
```

Comment:
So as far as I can see the breaking changes are:
- the workflows supplied all need to be updated so that condition functions must now be named `condition` and must be Graal.js compatible
- all downstream workers must be using a framework version that supports running Graal.js scripts